pysurfex/netcdf.py

```diff
@@ -984,8 +984,6 @@ def read_cryoclim_nc(infiles):
             grid_snow_class_read = ncf["classed_product"][:]
```

Comment:
Depending on cryoclim files used "classed_product" is not always the variable we use.